src/utils/azureClients.ts

```diff
@@ -43,3 +46,35 @@ export async function createAuthorizationManagementClient(context: AzExtClientCo
         return createAzureClient(context, (await import('@azure/arm-authorization')).AuthorizationManagementClient);
     }
 }
+
+/**
+ * @throws Throws an error if logsQueryClient is created by a user in a sovereign cloud environment.
```

Comment:
Should we throw? I feel like we should just return undefined and let the operation be skipped since this shouldn't be blocking the users from whatever command we're trying to run (probably deploy?)

Reply:
This is a tough one, I don't know.  I feel like we should throw, but I could go either way.

Reply:
I mean, if you're on a sovereign cloud, it would really suck for your deployment to always report a failure just because the log client isn't compatible.

It's not technically blocking the user, but I do think it'll be confusing. I think we can report it as a blue or gray node, saying skipped due to no availability in a sovereign cloud environment or something.